src/content/self-hosted/guides/okta/index.mdx

```diff
@@ -0,0 +1,161 @@
+
+First, update the auth section of your Helm configuration file (e.g. `values.yaml` ) with the values you obtained on the first step of this guide:
+
+```yaml
+auth:
+  openid:
+    enabled: true
+    clientId: $CLIENT_ID
+    clientSecret: $CLIENT_SECRET
+    endpoints:
+      issuer: $ISSUER_URL
+      authorization: $AUTHORIZATION_URL
+```
+
+In our demo instance, this looks like: 
+
+```yaml
+...
+auth:
+  opened:
+    enabled: true    
+    clientId: 0oa76rh0z5zgKG7XXXXX
+    clientSecret: dTgsfHbYV9tt91Ng4dzvpULw6y7kjJaYxzfXXXXX
+    endpoints:
+      issuer: https://dev-6093102.okta.com
+      authorization: https://dev-6093102.okta.com/oauth2/v1/authorize
+...
+```
+
+> You can also store the value of clientSecret [as a secret directly in Kubernetes](/docs/self-hosted/install/deployment/#cloud-provider-secret).
+
+Then, [upgrade your installation](/docs/self-hosted/install/upgrade/) to apply the changes. We recommend that you upgrade to the same version that you already have to minimize the changes and help with your troubleshooting. 
+
+Once the upgrade has been completed, go to your instance and click on the 'Log In' button to access your instance using Okta as the authentication provider.
+
+<p align="center"><Image src="/docs/self-hosted/guides/okta/images/with-okta.png" alt="Log in with Okta" /></p>
+
+## Customize the attributes that Okteto consumes
+
+By default, Okteto will use the attributes below when creating a user.
+
+- `Name` as the display name of the user.
+- `Nickname` as the username of the user.
+- `Email` as the email.
+- `Picture URL` for the user's profile picture.
+
+If you are not using some of those values, or you want to customize which attributes get mapped, you can customize that by using the `mapping` field inside the `openid` key in your Helm configuration file.
+
+```yaml
+auth:
+  openid:
+    enabled: true
+    clientId: $CLIENT_ID
+    clientSecret: $CLIENT_SECRET
+    endpoints:
+      issuer: $ISSUER_URL
+      authorization: $AUTHORIZATION_URL
+    mapping:
+      externalIDKey: nickname
+      nameKey: name
+      emailKey: email
+      pictureKey: picture
+```
+
+The value of every key in the `mapping` section must match a `claim` in your Okta + OpenID configuration, and it must contain a non-empty value. 
+
+For example, if you just want to use the `email` instead of the nickname for usernames, use the following configuration:
+
+```yaml
+auth:
+  opened:
+    enabled: true    
+    clientId: 0oa76rh0z5zgKG7XXXXX
+    clientSecret: dTgsfHbYV9tt91Ng4dzvpULw6y7kjJaYxzfXXXXX
+    endpoints:
+      issuer: https://dev-6093102.okta.com
+      authorization: https://dev-6093102.okta.com/oauth2/v1/authorize
+     mapping:
+      externalIDKey: email
+```
+
+## Troubleshooting
+
+### Log in fails with a 500 error
+
+This is most likely because the user was not added to the application. Make sure that the user (or group if you are using them) is assigned to the application that you just created.
```

Comment:
no need for that, it's kind of implied? 😅 

```suggestion
This is most likely because the user was not added to the application. Make sure that the user (or group) is assigned to the application that you just created.
```